### DISCUSSION TITLE
problem

### DISCUSSION THREAD

**Discussion by CoderTahmid** [github/india#1121] [Students]
<img width="382" alt="Screenshot 2023-12-18 221311" src="https://github.com/github/india/assets/154243953/5783bad3-2a6c-4cb0-8934-ea0ad0064120">
<img width="577" alt="Screenshot 2023-12-19 033639" src="https://github.com/github/india/assets/154243953/43fd4d2b-a9d5-4e3b-a39b-8d96be9ae839">

Why this code isn't running in my PC correctly?

**Comment by Himanshu8728** [comment]
The code logic is correct, but it doesn’t run properly due to two main reasons:

1. prompt() returns a string

prompt() always returns a string, while gameNum is a number.
So the comparison fails.

Fix: Convert the input to a number using Number().

2. prompt() works only in the browser

If this code is run in VS Code terminal or Node.js, it will not work because prompt() is a browser-only function.
It should be executed inside a browser using an HTML file.





let gameNum = 25;
let userNum = Number(prompt("Guess the game number : "));

while (userNum !== gameNum) {
  userNum = Number(prompt("You entered wrong number. Guess again : "));
}

console.log("Congratulations, you entered the right number");
